provision.sh

```diff
@@ -140,6 +149,9 @@ AddDefaultCharset UTF-8
     MIMEMagicFile conf/magic
 </IfModule>
 
+RewriteEngine On
+RewriteRule ^/((OC|GC)[A-Za-z0-9]{1,5})$ /searchplugin.php?userinput=$1 [NC]
```

Comment:
please change $ with \$ and $1 with \$1
after that move it please under ErrorDocument 404 entry.